### PR TITLE
Add delete link command (without parser)

### LINKED ISSUE #105: DeleteLink command

### PR DESCRIPTION
**Pull request by lavanyagarg112**
Fix #105 

Parser not implemented yet. But command has been implemented and thoroughly tested.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteLinkCommand.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        this.caregiverNric = caregiverNric;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        Person patient = model.getPerson(patientNric);
+        Person caregiver = model.getPerson(caregiverNric);
+        if (patient == null || caregiver == null) {
```
Comment (pradyuprasad) [inline src/main/java/seedu/address/logic/commands/DeleteLinkCommand.java:54]: You should use requireNonNull for this for consistency across the code

**Review by pradyuprasad — CHANGES_REQUESTED** [review]
Besides that looks good to me

File: src/main/java/seedu/address/logic/commands/DeleteLinkCommand.java (hunk 1 above)
Comment (lavanyagarg112) [reply src/main/java/seedu/address/logic/commands/DeleteLinkCommand.java]: Done. thanks for the review

**Review by pradyuprasad — APPROVED** [review]
Looks good to me. Will merge

**Pull request merged**